doc/tutorials/acrn-secure-boot-with-grub.rst

```diff
@@ -0,0 +1,177 @@
+.. _how-to-enable-acrn-secure-boot-with-grub:
+
+Enable ACRN Secure Boot with GRUB
+#################################
+
+This document shows how to enable ACRN secure boot with GRUB including:
+
+  -  ACRN Secure Boot Sequence
+  -  Generate GPG Key
+  -  Setup Standalone GRUB EFI Binary
+  -  UEFI Secure Boot
+
+  Validation Environment:
+	Hardware Platform: INTEL WHL
+	ACRN Scenario: Industry
+	Service VM: Ubuntu 18.04
+	GRUB: 2.02
+
+Note: Please make sure you have alternative ways to restore GRUB when you got stuck.
```

Comment:
I would rephrase this into something like: "Note that Grub may stop booting in case of problems, make sure you know how to recover a bootloader on your platform". @dbkinder , any better suggestion / (Not quite happy with what I wrote)

Reply:
Use `.. note::` here for proper rendering of the note.